### PR TITLE
add limit one log size feature

### PR DESCRIPTION
**Pull request by superJinQing**
add limit one log size feature

### CODE REVIEW

File: main.go [hunk 1]
```diff
@@ -20,11 +20,11 @@ import (
 
 const (
 	DefaultMultiCoreNum = 4
+	Path                = "./Log/"
```
Comment (dreamfly281) [inline main.go]: This const should be defined in Log module, as it is a part of log function, main function don't need care about this

File: net/net.go [hunk 2]
```diff
@@ -23,7 +23,5 @@ type Neter interface {
 
 func StartProtocol(pubKey *crypto.PubKey) protocol.Noder {
 	net := node.InitNode(pubKey)
-	net.ConnectSeeds()
```
Comment (dreamfly281) [inline net/net.go]: The function be called here to speed up the connection process, which means the initialization connection don't need wait 3 second until the update info timer be triggered

File: net/node/infoUpdate.go [hunk 3]
```diff
@@ -152,8 +151,8 @@ func (node *node) TryConnect() {
 	for _, n := range node.nbrNodes.List {
 		if n.GetState() == INACTIVITY && n.tryTimes < 3 {
 			//try to connect
-			node.reconnect(n)
 			n.tryTimes++
+			node.reconnect(n)
```
Comment (dreamfly281) [inline net/node/infoUpdate.go]: a bit confuse with this, if the reconnect be called, the trytimes will always be set to 0 as above funtion showed. 
so what the mean to do n.tryTimes++?

Comment (superJinQing) [inline net/node/infoUpdate.go]: If connected tryTimes should be 0, otherwise it should be return before set tryTimes be 0

**Pull request merged**